backend/tracim_backend/models/auth.py

```diff
@@ -118,7 +118,7 @@ class User(TrashableMixin, DeclarativeBase):
     MAX_IMPORTED_FROM_LENGTH = 32
     MAX_TIMEZONE_LENGTH = 32
     MIN_LANG_LENGTH = 2
-    MAX_LANG_LENGTH = 3
+    MAX_LANG_LENGTH = 5
```

Comment:
Ok so maybe I know, not sure though -> the max length of the languages id string 
Here it should be 6 since no_NO is 5 characters long. _Still not sure of the use of this variable_

Reply:
```suggestion
    MAX_LANG_LENGTH = 6
```

To check if the code language is valid